genesyscloud/resource_genesyscloud_auth_role.go

```diff
@@ -234,7 +234,7 @@ func readAuthRole(ctx context.Context, d *schema.ResourceData, meta interface{})
 	log.Printf("Reading role %s", d.Id())
 
 	return withRetriesForRead(ctx, d, func() *resource.RetryError {
-		role, resp, getErr := authAPI.GetAuthorizationRole(d.Id(), nil)
+		role, resp, getErr := authAPI.GetAuthorizationRole(d.Id(), false, nil)
```

Comment:
This is the new field call that will disable usercount.  This is an extremely expensive operation and we don't use if for CX as Code